repo/packages/H/hdfs/0/marathon.json

```diff
@@ -5,5 +5,15 @@
   "disk": 100,
   "instances": 1,
   "cmd": "{{#hdfs/custom-config}}echo {{hdfs/custom-config}} | base64 --decode > mesos-site.xml && {{/hdfs/custom-config}}cd hdfs-mesos* && if [ -e '../mesos-site.xml' ]; then cp -f ../mesos-site.xml ./etc/hadoop/; fi && ./bin/hdfs-mesos",
-  "uris": {{{hdfs/scheduler-uris}}}
+  "uris": {{{hdfs/scheduler-uris}}},
+  "healthChecks": [
+    {
+      "protocol": "COMMAND",
+      "command": { "value": "hadoop fs -ls hdfs://hdfs/ && hadoop fs -rm -f hdfs://hdfs/test1.txt && rm -f test2.txt && echo \"this is a test\" > test1.txt && hadoop fs -put test1.txt hdfs://hdfs/ && hadoop fs -get hdfs://hdfs/test1.txt test2.txt" },
```

Comment:
Looks like you are left with test2.txt in the sandbox.

Also, can we at least make the path for the test configurable and document this behaviour? The last thing we want to do is remove our customer's file.